core/src/components/tab-bar/tab-bar.scss

```diff
@@ -35,10 +35,18 @@
 
 :host(.ion-color),
 :host(.ion-color) ::slotted(ion-tab-button) {
-  background: #{current-color(base)};
   color: #{current-color(contrast, .7)};
 }
 
+:host(.ion-color),
+:host(.ion-color) ::slotted(ion-tab-button:not(.ion-focused)) {
```

Comment:
Is it possible to style the `ion-tab-button` one way and then add on `.ion-focused` styling? I always get nervous chaining `:not` because of the precedence it sets.